samples/todo-app/deployment.yaml

```diff
@@ -11,6 +11,8 @@ spec:
       labels:
         name: stats-cache
     spec:
+      nodeSelector:
+        kubernetes.io/os: linux
```

Comment:
I see all of them have `Linux`. At least one of them should be windows to test the mixed-OS scenarios robustly?